packages/typo3-docs-theme/resources/config/typo3-docs-theme.php

```diff
@@ -41,6 +44,10 @@
         ->set(IssueReferenceTextRole::class)
         ->tag('phpdoc.guides.parser.rst.text_role')
         ->set(phpDocumentor\Guides\ReferenceResolvers\Interlink\DefaultInventoryLoader::class)
+        ->set(Typo3InventoryRepository::class)
+        ->arg('$inventoryConfigs', param('phpdoc.guides.inventories'))
+        ->set(InventoryRepository::class, Typo3InventoryRepository::class)
+        ->arg('$inventoryConfigs', param('phpdoc.guides.inventories'))
```

Comment:
Hm, shouldn't line 47+48 be superfluous, and 49+50 alone suffice?

Reply:
Ack